src/internal/ClickAwayListener.js

```diff
@@ -21,6 +21,7 @@ export default class ClickAwayListener extends Component {
   };
 
   componentDidMount() {
+    this.isCurrentlyMounted = true;
```

Comment:
`Assignment to read-only properties is not allowed in strict mode` in IE11

`Uncaught TypeError: Cannot set property isMounted of #<ReactComponent> which has only a getter` in Chrome